extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/HiveTableCatalog.scala

```diff
@@ -103,6 +103,7 @@ class HiveTableCatalog(sparkSession: SparkSession)
       s"Require setting ${CATALOG_IMPLEMENTATION.key} to `hive` to enable hive support.")
     catalogName = name
     catalogOptions = options
+    sessionState.conf.setConfString("spark.sql.legacy.v1IdentifierNoCatalog", "true")
```

Comment:
this is only used in SessionCatalog, does it really affect v2 catalog code path ?

Reply:
Theoretically not, but HiveCatalog depends on the SessionCatalog...

Reply:
to be conservative, can we wrap all session catalog methods with:
```
withSQLConf("spark.sql.legacy.v1IdentifierNoCatalog" -> "true") {
  ...
}
```

I'm afraid it would affect session catalog behavior if people use both kyuubi hive catalog and vanilla spark session catalog.

Reply:
Good point, thanks